src/clients/Queue.js

```diff
@@ -302,6 +313,12 @@ export default class Queue extends Client {
     return this.request(this.createArtifact.entry, args);
   }
 
+  // tbd
```

Comment:
Does someone want to update the schema comments before we publish this? cc: @djmitche

Reply:
That'd be me.  I'll write something up now and open a PR in the Queue.

Reply:
https://github.com/taskcluster/taskcluster-queue/pull/207

Reply:
Awesome, thanks @jhford

Reply:
Schema description updated.